src/styles/sidebar/components/annotation-publish-control.scss

```diff
@@ -1,118 +1,72 @@
 @use "../../mixins/buttons";
 @use "../../mixins/forms";
 @use "../../mixins/layout";
+@use "../../mixins/utils";
 @use "../../variables" as var;
 
 .annotation-publish-control {
   @include layout.row;
+  @include layout.horizontal-rhythm(var.$layout-space);
 
-  // A split button with a primary submit on the left and a drop-down menu
-  // of related options to the right
-  .annotation-publish-control__btn {
-    @include layout.row;
-    $text-color: var.$color-text-inverted;
-    $default-background-color: var.$grey-mid;
-    $hover-background-color: var.$grey-6;
-    $h-padding: 9px;
-    $height: 35px;
-    $arrow-indicator-width: 26px;
-
-    height: $height;
-    position: relative;
-    margin-right: 1em;
+  &__cancel-button {
+    @include buttons.button--labeled;
+    // TODO: Magic number
+    padding: 0.75em;
+  }
+}
 
-    // Align the menu arrow correctly with the ▼ in the toggle
-    &-menu-arrow {
-      right: 5px;
-    }
+// A split button with a primary submit on the left and a drop-down menu
+// of related options to the right
+.annotation-publish-button {
+  @include layout.row;
+  // For proper menu alignment
+  position: relative;
 
-    // Make sure the menu content is wide enough to "reach" to the right-aligned
-    // menu arrow
-    &-menu-content {
-      min-width: 100%;
-    }
+  // Align the menu (upward) arrow correctly with the ▼ in the menu label icon
+  // Note the extra `&` needed for specificity against `Menu`'s own arrow styling
+  & &__menu-arrow {
```

Comment:
I'm a bit confused the the double `& &`

Reply:
Yeah, it's not the clearest, but it's a specificity hack. I should probably add a FIXME comment here because this is definitely not ideal.

This rule is nested within `.annotation-publish-button`. Without the extra `&`, the output-ted selector is:

`.annotation-publish-button__menu-arrow`

This has the same specificity as the rule in `Menu`, `.menu__toggle-arrow` and that rule would win out here—we need this rule to have more specificity than that one.

With the extra `&`, the output selector is

`.annotation-publish-button .annotation-publish-button__menu-arrow`

which wins the CSS specificity battle.

When compiled, `&` is output as the literal string value of the enclosing (immediate parent) selector, which in this case is `.annotation-publish-button`.

As background, this rule used to work here because there was a double nesting layer; all of `.annotation-publish-button` was nested inside of `.annotation-publish-control` (that means this selector used to be `.annotation-publish-control .annotation-publish-button__menu-arrow`, which had more specificity). However, our conventions ask us to avoid nesting more than one level deep and the nesting felt cumbersome in this module. I think there's more work to do here to find the sweet spot.